axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,43 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """
+    A player starts by always cooperating for the first 10 moves.
+    
+    The opponent answers are stored in the memory, whose maximum size is 10 results. From the tenth round on, the program analyzes the memory, and compare the number of defects and cooperates of the opponent, based in percentage. If cooperation occurs 30% more than defection, it will cooperate.
+    If defection occurs 30% more than cooperation, the program will defect. Otherwise, the program follows the TitForTat algorithm.
+    
```

Comment:
Can you add something like:

```
    Names

    - Short Mem: [Axelrod1980]_
    """
```

Except replace `Axelrod1980` by a key that you'd put in https://github.com/Axelrod-Python/Axelrod/blob/master/docs/reference/bibliography.rst